packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.tsx

```diff
@@ -39,61 +29,43 @@ export type CursorInfo = {
   after?: string;
 };
 
-export type TextEditorSlots = {
-  root?: Omit<ComponentProps<'div'>, 'ref'>;
-  editor?: {
-    className?: string;
-  };
-  content?: {
-    className?: string;
-  };
-};
-
 export type TextEditorProps = {
-  model: EditorModel; // TODO(burdon): Optional (e.g., just provide content if readonly).
-  readonly?: boolean; // TODO(burdon): Move into model.
+  dataTestId?: string; // TODO(burdon): Generalize?
```

Comment:
doesn't seem like this is used currently? what about having these props extend `div` props and forward any unknown props into the parent div? could omit `className` below then as well